shlr/zip/zlib/gzguts.h

```diff
@@ -21,7 +21,9 @@
 #include <stdio.h>
 #include "zlib.h"
 #include <string.h>
+#ifndef _MSC_VER
```

Comment:
I mean the other way :p. If it builds on windows without this include it should build without it on linux too. Try removing the block